Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
Hm. So I see QueryAppDetails is called many times for package-specific tasks, like killing, uninstalling, getting logs, etc. These are all unique to the package the developer is deploying/working on. Name is not unique enough for these tasks. (For example, there can be two Messenger apps installed.)

But PackageFullName isn't right either as it includes versioning information. If the user wants to deploy 0.2 and wipe 0.1, this would fail.

Instead the middle ground here is use of PackageFamilyName. This is unique to the package, sans versioning, platform, and PRAID info. Can be read from Identity/@Name in the appx manifest.